optimade/adapters/structures/aiida.py

```diff
@@ -60,12 +61,12 @@ def get_aiida_structure_data(optimade_structure: OptimadeStructure) -> Structure
                 symbols.append(chemical_symbol)
                 concentration.append(kind.concentration[index])
 
-        # AiiDA needs a definition for the mass, and for it to be > 0
-        # mass is OPTIONAL for OPTIMADE structures
-        mass = kind.mass if kind.mass else 1
+                # AiiDA needs a definition for the mass, and for it to be > 0
+                # mass is OPTIONAL for OPTIMADE structures
+                masses.append(kind.mass[index] if kind.mass else 1)
 
         structure.append_kind(
-            Kind(symbols=symbols, weights=concentration, mass=mass, name=kind.name)
+            Kind(symbols=symbols, weights=concentration, mass=masses, name=kind.name)
```

Comment:
Hi @CasperWA, AiiDA tests are the only ones failing now. What's the AiiDA way of handling the multiple masses per "kind"? Average mass?

Reply:
I'd say average mass, yeah. But I have to check to be completely sure.

Reply:
Right. So the AiiDA `Kind` is essentially equivalent to `ase.Atom` (not to be confused with the plural-names class `ase.Atoms`). This means it's not meant to have multiple masses, but merely a single mass. I'll try and update your PR accordingly.

Reply:
I have updated both the adapter for `aiida.orm.StructureData` and `ase.Atoms` to properly sum up the masses of an OPTIMADE species for each `Kind` or `Atom` (see 7f2a19f).

Reply:
(I'm currently creating the tests.)